src/app/server.settings/server.settings.component.html

```diff
@@ -108,7 +108,7 @@
                             </div>
                                 <div class="tab-pane container-fluid" id="users" >
                                     <div class="text-right" style="float: right" >
-                                        <button class="btn btn-fill btn-success" i18n="@@newUserButton"
+                                        <button *ngIf="userRole !== ReadOnlyUserType" class="btn btn-fill btn-success" i18n="@@newUserButton"
```

Comment:
I'm sorry where is the definition of 'ReadOnlyUserType'?

Reply:
It's defined in server.settings.component.ts.
https://github.com/ant-media/Ant-Media-Management-Console/blob/master/src/app/server.settings/server.settings.component.ts#L75